src/modules/commentPopup.js

```diff
@@ -0,0 +1,29 @@
+import { showComment, getComments, commentFormInput } from './comments.js';
+
+const popupContainer = document.querySelector('.popup-display');
+
+async function commentPopup(id, image, name, language, genres, rating, schedule) {
+  popupContainer.innerHTML = `
+    <div class="show-${id} show-image-container">
+      <div class="close-icon">X</div>
+      <img src="${image.original}" alt="" class="show-img" />
+      <h2 class="show-popup-title">${name}</h2>
+    </div>
```

Comment:
- [ ] Clicking on comments to show the comment popup for the first time is working as expected and when I close the popup and try to open it again the comment popup is not shown again until I restart the page (See the video below).
Please fix this problem.

https://www.loom.com/share/0454a7929fd3419488be61bf21b6e9a8